gammapy/cube/tests/test_basic_cube.py

```diff
@@ -0,0 +1,79 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import pytest
+import numpy as np
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from ...maps import MapAxis, WcsGeom, WcsNDMap
+from ...data import DataStore
+from ...cube.basic_cube import make_map_counts
+
+# initial time of run 110380 from cta 1DC
+t0_1DC = 664504199. * u.s
+
+energy_true = np.logspace(-2., 2., 30) * u.TeV
+energy_reco = np.logspace(-1., 1.5, 10) * u.TeV
+times = np.linspace(t0_1DC.value, t0_1DC.value+1900., 10)*u.s
+
+axis_energy_true = MapAxis(energy_true.value, interp='log', node_type='edge',
+                           name='energy_true', unit = energy_true.unit)
+axis_energy_reco = MapAxis(energy_reco.value, interp='log', node_type='edge',
+                           name='energy_reco', unit = energy_reco.unit)
+axis_time = MapAxis(times, node_type='edge', name='time',unit = times.unit)
+
+cta_1dc_store = "$GAMMAPY_EXTRA/datasets/cta-1dc/index/gps/"
+cta_1dc_runs = [110380]
+
+hess_crab_store = "$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/"
+hess_crab_runs = [23523]
+
+pos_crab = SkyCoord(83.633212, 22.01446, frame='icrs',unit='deg').galactic
+pos_cta = SkyCoord(0.0, 0.0, frame='galactic',unit='deg')
+
+geom_cta = {'binsz':0.02, 'coordsys':'GAL', 'width' : 15*u.deg,
+            'skydir' : pos_cta, 'axes' : [axis_energy_reco]}
+geom_cta_time = {'binsz':0.02, 'coordsys':'GAL', 'width' : 15*u.deg,
+                 'skydir' : pos_cta, 'axes' : [axis_energy_reco, axis_time]}
+
+geom_hess_true = {'binsz':0.02, 'coordsys':'GAL', 'width' : 10*u.deg,
+                  'skydir' : pos_crab, 'axes' : [axis_energy_true]}
+
+geom_hess_reco = {'binsz':0.02, 'coordsys':'GAL', 'width' : 10*u.deg,
+                  'skydir' : pos_crab, 'axes' : [axis_energy_reco]}
+
+@pytest.mark.parametrize("ds_path,run_list,geom",[(cta_1dc_store,cta_1dc_runs, geom_cta),
```

Comment:
I don't think there's any reason to parametrise over different event lists. Parametrisation over maps with different axes, yes that is very userful.

Reply:
OK, for count maps it is not very relevant indeed.